github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -119,3 +133,33 @@ enum HmacType {
 	SHA384 				= 3;
 	SHA512 				= 4;
 }
+ 
+// KeyType defines the key's type
+  enum KeyType {
+	KEY_TYPE_UNKNOWN 	= 0;
+	RSA2048 			= 1;
+	RSA3072 			= 2;
+	RSA4096 			= 3;
+	AES256_GCM96 		= 4;
+	ECDSA_P256 			= 5;
+	ECDSA_P384 			= 6;
+	ECDSA_P521			= 7;
+	HMAC_SHA2_224		= 8;
```

Comment:
Same here. This is an algorithm; really the key type is HMAC.

Reply:
ty!